saleor/payment/gateways/stripe/plugin.py

```diff
@@ -294,8 +302,13 @@ def confirm_payment(
             kind, action_required = self._get_transaction_details_for_stripe_status(
                 payment_intent.status
             )
+
+            if payment_intent.get("setup_future_usage"):
```

Comment:
Not sure if I understand. Is it required? I thought that stripe will return a payment_intent metadata so no need to copy them one more time. Am I missing something?